### ISSUE TITLE
sailboat can't detect action

### ISSUE THREAD

**Issue by nirav15** [DominusKelvin/sailboat#9]
sailboat can't detect action more than one hierarchy.

For example it will quick open api/login.js but can't open api/v-1/login.js.

**Comment by lennyAiko** [comment]
I also noticed it does not detect actions with hyphens.

For example `auth/refresh-token.js`

Maybe I should give this a shot. what do you think @DominusKelvin

**Comment by DominusKelvin** [comment]
Yeah but those deep nested actions I'm not even sure it's following proper action practices

**Comment by lennyAiko** [comment]
I concur to that
